docs/book/collaborate/profiles.md

```diff
@@ -0,0 +1,587 @@
+┠──────────────┼───────────────────────────────────────────────────────────────────────┨
+┃ STORE_URL    │ sqlite:////home/stefan/.config/zenml/profiles/sqlite_profile/zenml.db ┃
+┠──────────────┼───────────────────────────────────────────────────────────────────────┨
+┃ STORE_TYPE   │ sql                                                                   ┃
+┠──────────────┼───────────────────────────────────────────────────────────────────────┨
+┃ ACTIVE_STACK │ default                                                               ┃
+┠──────────────┼───────────────────────────────────────────────────────────────────────┨
+┃ ACTIVE_USER  │ default                                                               ┃
+┗━━━━━━━━━━━━━━┷━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
+```
+
+The location of the SQLite database can be customized during profile creation:
+
+```
+$ zenml profile create custom_sqlite -t sql --url=sqlite:////tmp/zenml/zenml_profile.db
+Running without an active repository root.
+Running with active profile: 'sqlite_profile' (global)
+Initializing profile custom_sqlite...
+Registering default stack...
+Registered stack with name 'default'.
+Profile 'custom_sqlite' successfully created.
+
+$ zenml profile set custom_sqlite
+Running without an active repository root.
+Running with active profile: 'sqlite_profile' (global)
+Active profile changed to: 'custom_sqlite'
+
+$ zenml profile describe
+Running without an active repository root.
+Running with active profile: 'custom_sqlite' (global)
+     'custom_sqlite' Profile Configuration (ACTIVE)     
+┏━━━━━━━━━━━━━━┯━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃ PROPERTY     │ VALUE                                 ┃
+┠──────────────┼───────────────────────────────────────┨
+┃ NAME         │ custom_sqlite                         ┃
+┠──────────────┼───────────────────────────────────────┨
+┃ STORE_URL    │ sqlite:////tmp/zenml/zenml_profile.db ┃
+┠──────────────┼───────────────────────────────────────┨
+┃ STORE_TYPE   │ sql                                   ┃
+┠──────────────┼───────────────────────────────────────┨
+┃ ACTIVE_STACK │ default                               ┃
+┠──────────────┼───────────────────────────────────────┨
+┃ ACTIVE_USER  │ default                               ┃
+┗━━━━━━━━━━━━━━┷━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
+```
+
+#### MySQL Profile
+
+To connect the ZenML Profile to an existing MySQL database, some additional
+configuration is required on the MySQL server to create a user and database:
+
+```
+mysql -u root
+GRANT ALL PRIVILEGES ON *.* TO 'zenml'@'%' IDENTIFIED BY 'password';
+
+mysql -u zenml -p
+CREATE DATABASE zenml;
+```
+
+Then, on the client machine, some additional packages need to be installed (check
+[the SQLAlchemy documentation](https://docs.sqlalchemy.org/en/14/dialects/mysql.html)
+for the various MySQL drivers that are supported and how to use them):
+
+```
+sudo apt install libmysqlclient-dev
```

Comment:
Is it worth making a comment about how this applies to Debian Linux and other OSes must deal with it in their custom ways?

Reply:
Yes !